tests/wpt/include.ini

```diff
@@ -5,6 +5,8 @@ skip: true
   skip: false
 [2dcontext]
   skip: false
+[bluetooth]
+  skip: false
```

Comment:
Let's remove this. The test expectations won't match when run in the non-servodriver test runs.

Reply:
Done